pkg/koordlet/runtimehooks/protocol/container_context.go

```diff
@@ -181,8 +181,9 @@ func (c *ContainerRequest) FromReconciler(podMeta *statesinformer.PodMeta, conta
 }
 
 type ContainerResponse struct {
-	Resources        Resources
-	AddContainerEnvs map[string]string
+	Resources          Resources
+	AddContainerEnvs   map[string]string
+	AddContainerMounts []*api.Mount
```

Comment:
We'd better avoid referring to NRI API directly in the koordinator protocol. Consider making a new Mount struct and we can transform the internal protocol to the NRI protocol or the CRI protocol.